src/lib/slide-toggle/slide-toggle.ts

```diff
@@ -213,7 +213,7 @@ export class MdSlideToggle implements AfterContentInit, ControlValueAccessor {
 
 
   /** TODO: internal */
-  _onDragStart() {
+  _onDragStart(event: Event) {

```

Comment:
good idea, removed the parameter at the callsite instead.

On Wed, Aug 24, 2016 at 12:58 PM Paul Gschwendtner <notifications@github.com>
wrote:

> In src/lib/slide-toggle/slide-toggle.ts
> <https://github.com/angular/material2/pull/1113#discussion_r76127308>:
>
> > @@ -213,7 +213,7 @@ export class MdSlideToggle implements AfterContentInit, ControlValueAccessor {
> >
> >
> >    /** TODO: internal */
> > -  _onDragStart() {
> > +  _onDragStart(event: Event) {
>
>
>    - Why don't we remove the parameters from the HTML instead (This does
>    not show any linter warnings / infos etc.)
>    - Or make it consistent with the onDrag function, by using HammerInput
>    instead? (to get proper autocompletion)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/angular/material2/pull/1113/files/1b317155f01c442ddea69907006dca011d73392c#r76127308>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAC5IxzZ79qWPomEoy50jO6QDXmcEld0ks5qjKJagaJpZM4JsYxG>
> .
>